pyccel/codegen/printing/ccode.py

```diff
@@ -1005,7 +1005,10 @@ def _print_Import(self, expr):
                               f'#define i_key {container_key}',
                               f'#include "{stc_name + "/" + container[0]}.h"',
                               '#endif\n'))
-
+        elif source.startswith('Set_pop'):
+            _ , i_type, i_key = source.split('/')
+            return '\n'.join((f'#define i_type {i_type}',
+                   f'#define i_key {i_key}\n',))
```

Comment:
Is this sufficient? You don't actually need to import anything? Please leave a comment on this PR with an example code and its translation

Reply:
`#include "STC_Extensions/Set_extensions`

This is the line that I was expecting to see here but don't

Reply:
Don't you need a scope guard for this?

Reply:
Fixed : 8243ccb